src/main/java/org/jboss/ejb/client/ConfigurationBasedEJBClientContextSelector.java

```diff
@@ -197,16 +246,17 @@ private static void parseInterceptorType(final ConfigurationXMLStreamReader stre
         } else {
             cl = ConfigurationBasedEJBClientContextSelector.class.getClassLoader();
         }
-        final Class<? extends EJBClientInterceptor> interceptorClass;
+        final Class<?> clazz;
```

Comment:
`Class<? extends T>` would be better here I think.

Reply:
Ups, you're right, I missed that one...